internal/host/statemachine.go

```diff
@@ -452,7 +452,7 @@ func NewHostStateMachine(sm stateswitch.StateMachine, th TransitionHandler) stat
 	// Prepare for installation
 	sm.AddTransitionRule(stateswitch.TransitionRule{
 		TransitionType: TransitionTypeRefresh,
-		Condition:      stateswitch.And(If(ValidRoleForInstallation), If(IsConnected), If(IsMediaConnected), If(ClusterPreparingForInstallation)),
+		Condition:      stateswitch.And(If(ValidRoleForInstallation), If(IsConnected), If(IsMediaConnected), stateswitch.Or(If(ClusterPreparingForInstallation), th.IsDay2Host)),
```

Comment:
will not it cause node to move to HostStatusPreparingForInstallation right after it moved to HostStatusKnown ?
Without actually waiting to user to click install?

Reply:
Oh yes good point, how do we normally detect that a user wants to install a day 2 host? Before we also just moved from `known` to `installing` and only condition is `th.IsDay2Host` 😮 
https://github.com/openshift/assisted-service/pull/6143/files#diff-d26dc3c3c847de4b476ad1917cd7f5565935d06b6804e53d576ea04f26bc414bL288-L300